README.md

```diff
@@ -45,35 +45,51 @@ git clone https://github.com/Azure-Samples/functions-quickstart-python-azd.git
 ```
 
 ### Create a virtual environment
-1) Open the terminal, navigate to the project folder, and run the following commands:
+Open the terminal, navigate to the project folder, and run the following commands:
 
+#### bash
```

Comment:
I think that my original docs on this are incorrect and the tabs should be `Windows | Linux/macOs` but I'll fix this when update readmes (and in the regular Python quickstart too).